### ISSUE TITLE
Expand localization of distance/user preference across the app

### ISSUE THREAD

**Issue by ErikGraciosa** [pinballmap/pbm-react#535]
Found that the events page text when no events were returned was not toggling km/mi but the buttons to change distance do toggle km/mi. Looking across the app, the activities page does not toggle km/mi.

**Comment by ErikGraciosa** [comment]
Added a draft PR. https://github.com/pinballmap/pbm-react/pull/536
The distances I added in km are rounding to the nearest nice looking number. It didn't look like the API used a unit currently so the distance change is only presentational and the actual distance is still miles. Probably could refactor this some to reduce the repetition of values but curious if the change is desired or not before getting too far into it.

**Comment by RyanTG** [comment]
Yeah, I think we just didn't bother with adjusting the distance unit in this case because it didn't seem terribly important. Beth can probably refresh my memory, though.

So I guess the options are:
- Keep as is
- Show km or mi but don't actually change the distance value used in the api requests
- Show km or mi but change the distance value in the api requests

**Comment by ErikGraciosa** [comment]
I would also add

- Fix settings page to match units.
![image](https://github.com/pinballmap/pbm-react/assets/70562690/0aa1e565-7167-412d-9897-2bda3c04e22d)

**Comment by RyanTG** [comment]
This PR is open but incomplete. Are you planning to complete it?

**Comment by ErikGraciosa** [comment]
Yes I can promote this from draft. I'll be able to submit in the next few days. I haven't loaded the app locally in a while, I'll likely need a new build. Is there a place to get that from? @RyanTG

**Comment by RyanTG** [comment]
I'll make new dev builds. Can you shoot an email to admin@pinballmap and then I'll send you the link to it?

**Comment by ErikGraciosa** [comment]
sent.

**Comment by RyanTG** [comment]
Text now shows correct units. https://github.com/pinballmap/pbm-react/commit/16be8c11dc6caa271c26529c3e99759fb534b791 https://github.com/pinballmap/pbm-react/commit/f9553cd888a94d12b2eed1f3f8bd97465e9f8651
